build/test.py

```diff
@@ -59,6 +59,34 @@ def __call__(self, parser, namespace, new_argument, option_string=None):
     merged[key] = value
     setattr(namespace, self.dest, merged)
 
+class _KeyValidatorType():
```

Comment:
How about _KeyValueValidator as a function name?  (No need for a class.  See my other comments.)

Reply:
good idea

Reply:
done!